tutorials/setup.md

```diff
@@ -109,11 +106,16 @@ To add build tags to the make option the `BUILDTAGS` variable must be set.
 make BUILDTAGS='seccomp apparmor'
 ```
 
-| Build Tag | Feature                            | Dependency  |
-|-----------|------------------------------------|-------------|
-| seccomp   | syscall filtering                  | libseccomp  |
-| selinux   | selinux process and mount labeling | libselinux  |
-| apparmor  | apparmor profile support           | <none>      |
+| Build Tag                        | Feature                                         | Dependency   |
+|----------------------------------|-------------------------------------------------|--------------|
+| seccomp                          | syscall filtering                               | libseccomp   |
+| selinux                          | selinux process and mount labeling              | libselinux   |
+| apparmor                         | apparmor profile support                        | <none>       |
+| exclude_graphdriver_btrfs        | exclude btrfs as a storage option               | <none>       |
+| btrfs_noversion                  | for building btrfs version < 3.16.1             | btrfs        |
+| exclude_graphdriver_devicemapper | exclude devicemapper as a storage option        | <none>       |
+| libdm_no_deferred_remove         | don't compile deferred remove with devicemapper | devicemapper |
```

Comment:
We could also add the build tags for containers/image or refer to them: https://github.com/containers/image#supported-build-tags

Reply:
I sorted those and the ones for c/storage into their own table!